### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by KrzysiekJ**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<http://wolontariat.uw.edu.pl/>`__. Modules related to these websites
+are marked with suffixes ``_fpbz`` and ``_ucw``, respectively.
+
+Installation
+------------
+
+Generally GooDoo is based on Odoo 8.0 compatible modules, however
+there are some constraints on which version of Odoo can be
+used. `Commit 2067a20 <https://github.com/odoo/odoo/commit/2067a20>`__
+introduced a regression in GooDoo, therefore an earlier version of
```
Comment (ludwiktrammer) [inline README.rst]: Could you link "introduced a regression" to [the bug report](https://github.com/odoo/odoo/issues/5319) so it is possible to check the details of the problem and to know if it gets fixed in the future?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+======
+
+GooDoo (codename: BestJa) is an utility for professional management of
+volunteers’ work.
+
+There are two websites for which GooDoo has been created, owned by
+`Federacja Polskich Banków Żywności <http://bankizywnosci.pl/>`__ and
+`Uniwersyteckie Centrum Wolontariatu
+<http://wolontariat.uw.edu.pl/>`__. Modules related to these websites
+are marked with suffixes ``_fpbz`` and ``_ucw``, respectively.
```
Comment (ludwiktrammer) [inline README.rst]: We could also add an information that modules that are not prefixed with "bestja_" may be used outside of the Bestja ecosystem, as a standalone modules in any Odoo installation.

Comment (KrzysiekJ) [inline README.rst]: Fixed.

**Comment by ludwiktrammer** [comment]
Looks ok to me :+1:

**Pull request merged**